ThreeBotPackages/threefold/communityapp/CommunityFactory.py

```
@@ -3,11 +3,19 @@
 
 class CommunityFactory(j.baseclasses.threebot_factory):
 
-    __jslocation__ = "j.threebot.package.communiy_bot"
+    __jslocation__ = "j.threebot.package.community_bot"
+
+    def _get_admin_referral(self):
+        bcdb = j.data.bcdb.get("tf_community_app")
+        model = bcdb.model_get(url="threefold.community.user.1")
+        users = model.find(name="admin")
+        if users:
+            return users[0].referral_code
+        return "Sorry , admin not created yet"
```

Please raise a NotFound error with this message instead.